tests/PSRule.Rules.Azure.Tests/Azure.Baseline.Tests.ps1

```diff
@@ -96,5 +96,12 @@ Describe 'Baselines' -Tag Baseline {
             $filteredResult | Should -Not -BeNullOrEmpty;
             $filteredResult.Length | Should -Be 226;
         }
+
+        It 'With Azure.GA_2021_12' {
```

Comment:
@ArmaanMcleod The baseline won't exist until December. Happy to discuss further but I don't think we need a test here until this baseline is released in the future. Otherwise, we'll be bumping the number of rules each PR.

Happy for you to remove `With Azure.GA_2021_12` test until then.

Reply:
@BernieWhite No worries, removed that test 🙂. Thought it would save effort needing to add it later but happy to avoid bumping count on each new PR 😄.